vendor.conf

```diff
@@ -1,6 +1,7 @@
 # cri dependencies
 github.com/docker/docker                            4634ce647cf2ce2c6031129ccd109e557244986f
-github.com/opencontainers/selinux                   v1.5.1
+github.com/opencontainers/selinux                   v1.6.0
+github.com/willf/bitset								d5bec3311243426a3c6d1b7a795f24b17c686dbb # 1.1.10+ used by selinux pkg
```

Comment:
This looks like it is using tabs where the rest of the file separates with spaces. This should also sort below `tchap`